app/frontend/actions.ts

```diff
@@ -139,7 +139,7 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
           await loadWasmModule()
           const cryptoProvider = ShelleyJsCryptoProvider({
             walletSecretDef,
-            network: NETWORKS.SHELLEY.INCENTIVIZED_TESTNET,
+            network: NETWORKS.SHELLEY.MAINNET,
```

Comment:
I would make this configurable on .env level - we need a quick way to switch between networks once mainnet becomes live